fontbe/src/paths.rs

```diff
@@ -11,25 +11,26 @@ pub struct Paths {
     build_dir: PathBuf,
     glyph_dir: PathBuf,
     debug_dir: PathBuf,
-    output_file: Option<PathBuf>,
+    output_file: PathBuf,
 }
 
 impl Paths {
     pub fn new(build_dir: &Path) -> Paths {
         let glyph_dir = build_dir.join("glyphs");
         let debug_dir = build_dir.join("debug");
         let build_dir = build_dir.to_path_buf();
+        let output_file = build_dir.join("font.ttf");
         Paths {
             build_dir,
             glyph_dir,
             debug_dir,
-            output_file: None,
+            output_file,
         }
     }
 
     pub fn with_output_file(build_dir: &Path, output_file: &Path) -> Paths {
```

Comment:
looks like this is now unused.. clippy doesn't complain because it's pub but I think this is a remnant of the old approach and might be deleted